build.cmd

```diff
@@ -1,7 +1,7 @@
 @echo off
 cls
 
-SET BUILD_DIR=build
+SET BUILD_DIR=%cd%\build
```

Comment:
Should we better use `%~dp0` instead?

```suggestion
SET BUILD_DIR=%~dp0\build
```
That way we are never affected by the current working directory.